data/strings/strings.txt

```diff
@@ -28817,6 +28819,7 @@
     tags = android
     en = https://organicmaps.app/faq/text-to-speech-android-tts/
     es = https://organicmaps.app/es/faq/instrucciones-de-voz-tts-en-android/
+    fr = https://organicmaps.app/fr/faq/synthèse-vocale-tts-sur-android
```

Comment:
This link is invalid. English page https://organicmaps.app/faq/text-to-speech-android-tts/ is not yet translated to French. You can leave English page link or https://organicmaps.app/fr/faq/voice/cant-hear-instructions/

Reply:
Maybe it's better to translate the related FAQ page to French and then have the good link? )

https://github.com/organicmaps/organicmaps.github.io/tree/master/content/faq/voice/text-to-speech-android-tts

Reply:
> This link is invalid. English page https://organicmaps.app/faq/text-to-speech-android-tts/ is not yet translated to French. You can leave English page link or https://organicmaps.app/fr/faq/voice/cant-hear-instructions/

https://github.com/organicmaps/organicmaps.github.io/pull/250